src/applications/representatives/components/PoaRequestsTable/PoaRequestsTable.jsx

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { acceptDeclinePOARequest } from '../../actions/poaRequests';
+
+const PoaRequestsTable = ({ poaRequests }) => {
+  return (
+    <va-table data-testid="poa-requests-table" sort-column={1}>
+      <va-table-row slot="headers">
+        <span>Claimant</span>
+        <span>Submitted</span>
+        <span>Description</span>
+        <span>Status</span>
+        <span>Actions</span>
+      </va-table-row>
+      {poaRequests.map(({ id, name, date, description, status }) => (
+        <va-table-row key={id}>
+          <span data-testid={`${id}-claimant`}>{name}</span>
+          <span data-testid={`${id}-submitted`}>{date}</span>
+          <span data-testid={`${id}-description`}>{description}</span>
+          <span data-testid={`${id}-status`}>{status}</span>
+          <span>
+            {status === 'Pending' && (
```

Comment:
Is there a way we can help the caller deal only with known, discrete entities here rather than needing to know which string values to check for? One example would be to have an object that has a method `isPending` and no public access to the underlying string field. I could also see a more functional version that takes our raw data as an argument so that it's necessary to map over every row of data we get back in preparation first. I'm not so well-versed with the best patterns, but I think it could be worthwhile to express this basic design principle in the code early so that people are encouraged to continue following it later, even if we end up preferring some other form.

Reply:
FWIW, stuff is so early, that pursuing this code organization suggestion right now might validly be worth punting on.

Reply:
@williamphelps13 I like the intermediary solution to express the encapsulation just locally for now. Since this function does not depend on the lifecycle of the React component, is it best practice to place it outside of the component's scope?